<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>photoshow</class>
 <widget class="QWidget" name="photoshow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/tu/Res/MainWindow/blackbg.png);</string>
  </property>
  <widget class="QLabel" name="m_pLb_PhotoShow">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>920</width>
     <height>811</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLb_FileIndTotal">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color: white;           
    background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLb_FileName">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>351</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color: white;           
    background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="label_left">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>80</width>
     <height>146</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tu/Res/btn_left.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="label_right">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>300</y>
     <width>80</width>
     <height>146</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tu/Res/btn_right.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="label_return">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>700</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tu/Res/return.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="label_delete">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>680</y>
     <width>61</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tu/Res/delfile.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
